csharp/src/Google.Protobuf/Collections/RepeatedField.cs

```diff
@@ -300,10 +296,7 @@ public bool Remove(T item)
         /// <param name="values">The values to add to this collection.</param>
         public void Add(RepeatedField<T> values)
         {
```

Comment:
Removed the overload for Add, and put the code in AddRange instead. Going through CI now...